### DISCUSSION TITLE
Redis-To-ELK

### DISCUSSION THREAD

**Discussion by oencarnacion** [certtools/intelmq#2226] [General]
Hi,

@sebix  I would like to know if on any occasion you have been able to index the feeds coming from Shadowserver to ELK converting the data from String to Geopoint Type for the creation of a map. Currently I am trying it with logstash but it does not work for me, below I show you my logstash configuration file:

![image](https://user-images.githubusercontent.com/87453279/154273928-5496d1a3-a91c-4ecf-9efb-e1997fe1dd01.png)

![image](https://user-images.githubusercontent.com/87453279/154273877-faa1de5c-4b58-43cd-be0b-1c27e4b17b18.png)

**Comment by sebix** [comment]
No, sorry. I don't yet have experience with these components.
